code42_connector.py

```diff
@@ -103,8 +103,8 @@ def _handle_remove_departing_employee(self, param):
         action_result = self._add_action_result(param)
         username = param["username"]
         user_id = self._get_user_id(username)
-        response = self.client.detectionlists.departing_employee.remove(user_id)
-        action_result.add_data(response.data)
+        self.client.detectionlists.departing_employee.remove(user_id)
+        action_result.add_data({"userId": user_id})
```

Comment:
In the JSON file, we were saying that we were outputting the ID when we were not in fact.. Also, remember the username is already included in output and can be used in subsequent actions in a playbook since it was a parameter and that happens by default.